kore/src/Kore/Repl/Data.hs

```diff
@@ -342,15 +328,19 @@ helpText =
     \<alias>                               runs an existing alias\n\
     \load file                             loads the file as a repl script\n\
     \proof-status                          shows status for each claim\n\
-    \log <severity> [<scope>] <type>       configures the logging output\n\
+    \log <severity> [<entry>] <type>       configures the logging output\n\
```

Comment:
As @yzhang90 pointed out, the syntax here is confusing. We use `[]` to mark a list of elements here, but we use `[]` to mark optional arguments in other commands. Not sure what a good solution is though.

Reply:
`"[" <entry> ... "]"`?